content/backend/graphql-js/3-a-simple-mutation.md

```diff
@@ -128,14 +128,14 @@ const resolvers = {
 
 </Instruction>
 
-First off, note that you're entirely removing the `Link` resolvers (as explained before). They are not needed because the GraphQL server infers what they look like.
+First, note that the Link resolvers can be removed entirely and maintain the same functionality (as explained in the last section **The query resolution process**). They are not needed because GraphQLServer infers what they look like as the follow a general format.
```

Comment:
I like the old way here too, just liked how you added the name of the section... Could it be just:

```
First off, note that you're entirely removing the `Link` resolvers (as explained in the last section **The query resolution process**). They are not needed because the GraphQL server infers what they look like.	
```